token/services/db/driver/audit.go

```diff
@@ -49,9 +50,4 @@ type AuditDBDriver interface {
 	Open(cp ConfigProvider, tmsID token2.TMSID) (AuditTransactionDB, error)
 }
 
-type ConfigProvider interface {
-	UnmarshalKey(key string, rawVal interface{}) error
-	GetString(key string) string
-	IsSet(key string) bool
-	TranslatePath(path string) string
-}
+type ConfigProvider = core.ConfigProvider
```

Comment:
I think it is better to not link to core because it does not guarantee that this interface will remain there